config/clock.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'clockwork'
+require 'active_support/time'
+
+def execute_rake_task(task)
+  `bundle exec rake #{task}`
+end
+
+module Clockwork
+  every(1.hour, 'withdrawals:run') { execute_rake_task('smartvpn:withdrawals:run') }
```

Comment:
And what it we spawn sidekiq jobs instead of rake tasks? So we can move rake task code into sidekiq worker.
WDYT, @ilyaerin ?

Reply:
Yes, it's a good idea. Should these changes be in this PR?

Reply:
yep, why not?

Reply:
Ok. Got it.